app/src/test/java/at/aau/edu/wizards/gameModel/GameModelDealerUnitTest.kt

```diff
@@ -87,6 +88,7 @@ class GameModelDealerUnitTest {
         return amount
     }
 
+    // FIXME 4 methods duplicate code
```

Comment:
Yeah, i can prob collapse these by giving the type as a function parameter.
Good input!

Reply:
done